-1

**Failed Tests**:   Build 
**Summary**: https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-e5201a/13513/summary.html
**COMMIT**: 452e859b2bfe280429531a7cf5226de7bbd68b39
**CMSSW**: CMSSW_11_3_X_2021-03-14-2300/slc7_amd64_gcc900
**User test area**: For local testing, you can use `/cvmfs/cms-ci.cern.ch/week0/cms-sw/cmssw/33179/13513/install.sh` to create a dev area with all the needed externals and cmssw changes.


## Build

I found compilation error when building: 

<pre>>> Compiling edm plugin /data/cmsbld/jenkins/workspace/ib-run-pr-tests/CMSSW_11_3_X_2021-03-14-2300/src/JetMETCorrections/FFTJetModules/plugins/FFTJetCorrectorDBReader.cc
>> Compiling edm plugin /data/cmsbld/jenkins/workspace/ib-run-pr-tests/CMSSW_11_3_X_2021-03-14-2300/src/JetMETCorrections/FFTJetModules/plugins/FFTJetCorrectorDBWriter.cc
>> Compiling edm plugin /data/cmsbld/jenkins/workspace/ib-run-pr-tests/CMSSW_11_3_X_2021-03-14-2300/src/JetMETCorrections/FFTJetModules/plugins/FFTJetLookupTableESProducer.cc
>> Compiling edm plugin /data/cmsbld/jenkins/workspace/ib-run-pr-tests/CMSSW_11_3_X_2021-03-14-2300/src/JetMETCorrections/FFTJetModules/plugins/FFTJetSealModule.cc
In file included from /data/cmsbld/jenkins/workspace/ib-run-pr-tests/CMSSW_11_3_X_2021-03-14-2300/src/JetMETCorrections/FFTJetModules/plugins/FFTJetSealModule.cc:8:
/data/cmsbld/jenkins/workspace/ib-run-pr-tests/CMSSW_11_3_X_2021-03-14-2300/poison/CondFormats/JetMETObjects/interface/FFTJet.h:1:2: error: #error THIS FILE HAS BEEN REMOVED FROM THE PACKAGE.
    1 | #error THIS FILE HAS BEEN REMOVED FROM THE PACKAGE.
      |  ^~~~~
In file included from /cvmfs/cms-ib.cern.ch/nweek-02672/slc7_amd64_gcc900/external/boost/1.72.0-ljfedo2/include/boost/serialization/extended_type_info_typeid.hpp:37,
                 from /cvmfs/cms-ib.cern.ch/nweek-02672/slc7_amd64_gcc900/external/boost/1.72.0-ljfedo2/include/boost/archive/detail/iserializer.hpp:45,
                 from /cvmfs/cms-ib.cern.ch/nweek-02672/slc7_amd64_gcc900/external/boost/1.72.0-ljfedo2/include/boost/archive/detail/interface_iarchive.hpp:22,
</pre>

--- a/JetMETCorrections/FFTJetModules/plugins/FFTJetSealModule.cc
+++ b/JetMETCorrections/FFTJetModules/plugins/FFTJetSealModule.cc
@@ -4,8 +4,7 @@
 #include "JetMETCorrections/FFTJetObjects/interface/FFTJetCorrectorSequenceRcdTypes.h"
 #include "JetMETCorrections/FFTJetObjects/interface/FFTJetLookupTableRcdTypes.h"
 #include "JetMETCorrections/FFTJetObjects/interface/FFTJetLookupTableSequence.h"
-
-#include "CondFormats/JetMETObjects/interface/FFTJet.h"
+#include "JetMETCorrections/FFTJetObjects/interface/FFTJet.h"
 
 REGISTER_PLUGIN(FFTBasicJetCorrectorSequenceRcd, FFTBasicJetCorrectorSequence);
 REGISTER_PLUGIN(FFTGenJetCorrectorSequenceRcd, FFTGenJetCorrectorSequence);

